pkg/cluster/config/types.go

```diff
@@ -43,6 +43,12 @@ type Node struct {
 	// Role defines the role of the nodw in the in the Kubernetes cluster managed by `kind`
 	// Defaults to "control-plane"
 	Role NodeRole
+	// CNI is the name of the desired network plugin
+	CNI string
```

Comment:
if we are going to add these they should be under a CNI object and here it should be a pointer so that we can use omitifempty with conjunction with the types.go from v1alpha2

Reply:
That's a good idea. So something like?
type CNI struct{
CNIType string 
CNIConfigFileName string
CNIConfig []string
}